src/components/TaskCharts.js

```diff
@@ -0,0 +1,105 @@
+import _ from 'lodash';
+import { Component, PropTypes } from 'react';
+import Task from './Task';
+import { createTaskChart } from './../charts/createTaskChart';
+import rd3 from 'react-d3-library';
+
+/**
+Notes:
+Adjusted inline CSS to match design specs, which results in non centered visualizations.  Can adjust.
+**/
+
+/**
+Import React Components from RD3 library to convert D3 into React Elements,
+otherwise virtual dom will be effected negatively, and diffing algo won't work.
+**/
+
+const CompleteComponent = rd3.Component;
+const InProgressComponent = rd3.Component;
+const ToDoComponent = rd3.Component;
+
+class TaskCharts extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {completeData: '', inProgressData: '', toDoData: ''};
```

Comment:
Is there any particular reason you chose to give this internal state based on props and update state when receiving new props?
Just wondering if there is an advantage to doing it this way rather than just directly having the graph components based on the props.